tap_github/organization_streams.py

```diff
@@ -0,0 +1,168 @@
+"""User Stream types classes for tap-github."""
+
+from typing import Dict, List, Optional, Iterable, Any
+
+from singer_sdk import typing as th  # JSON Schema typing helpers
+
+from tap_github.client import GitHubRestStream
+
+
+class OrganizationStream(GitHubRestStream):
+    name = "organizations"
+
+    @property
+    def path(self) -> str:  # type: ignore
+        """Return the API endpoint path."""
+        return "/orgs/{org}"
+
+    @property
+    def partitions(self) -> Optional[List[Dict]]:
+        """Return a list of partitions."""
+        return [{"org": org} for org in self.config["organizations"]]
+
+    def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
```

Comment:
I think `dict` as a type hint only works for python 3.9+. Before that you need `Dict` (same story for `List`, etc...)(https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#built-in-types).